Maple2.Server.Web/Endpoints/Upload.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Maple2.Model.Enum;
+using Maple2.PacketLib.Tools;
+using Maple2.Server.Web.Constants;
+using Microsoft.AspNetCore.Http;
+using Serilog;
+
+namespace Maple2.Server.Web.Endpoints;
+
+public static class UploadEndpoint {
+    public static async Task<IResult> Post(HttpRequest request) {
+        Stream bodyStream = request.Body;
+
+        MemoryStream memoryStream = await CopyStream(bodyStream);
+        if (memoryStream.Length == 0) {
```

Comment:
why not check `bodyStream.Length` directly?

Reply:
yeah it's strange. I get:
`Failed to read field: Value does not fall the expected range.`
i've tried reading length on request.Body too.